examples/getting_started/fatigable_pendulum.py

```diff
@@ -0,0 +1,263 @@
+from math import pi
```

Comment:
This file should be called something like `example_fatigue_tau` 
And as long as it is not part of the `bioptim` API (that is, it is implemented using custom), I would put that example in the `torque_driven_ocp` folder

Reply:
Done